memgpt/config.py

```diff
@@ -192,10 +240,57 @@ def save(self):
             self.anon_clientid = self.generate_uuid()
         set_field(config, "client", "anon_clientid", self.anon_clientid)
 
-        if not os.path.exists(MEMGPT_DIR):
-            os.makedirs(MEMGPT_DIR, exist_ok=True)
+        # logging
```

Comment:
@sarahwooders wondering should we potential write these out to a different file called logging.config or something?

Reply:
@cpacker It is possible, but having multiple configuration files might create confusion. There are only a few config options. Best practices include having a single configuration file that works within the existing config. Is there a reason it should be separated?

Reply:
Sorry just saw this message - the `~/.memgpt/config` file is quite annoying to maintain, since every time we change it we have to figure out how we will handle old config versions. Also, we are planning to use the `MemGPTConfig` object as basically user settings for the MemGPT server, which wont need any logging information. Given that only developers will be making modifications to log settings, I think we should keep log settings outside of  the `MemGPTConfig` class.

Reply:
@sarahwooders I can change it to code, but I will have to abandon this pull request and delete this branch. I will redo it in a new branch and pull the request. Or change to "logging.config" as the config file.  I hate having multiple configuration files for reasons like configuring Spring Boot for Java web applications. There are so many config files. Setting it up properly requires finding the correct file for the configuration option you need to change. It is still better to load from a configuration rather than change source files to change.  This is better for maintaining code stability and keeping setting out of the code.  This will prevent you from accidentally committing debug code to production.  This comes from my DevOps Engineering experience and accidentally dealing with bad releases from developers.